daffodil-sapi/src/main/scala/org/apache/daffodil/sapi/Daffodil.scala

```diff
@@ -138,11 +138,11 @@ class Compiler private[sapi] () {
    * Reload a saved parser from a file
    *
    * To allow jar-file packaging, (where the savedParser might be part of a jar),
-   * it is recommended to use the other version of [[Compiler#reload(savedParser:java\.nio\.channels\.ReadableByteChannel)*]] where the argument is
+   * it is recommended to use the other version of [[Compiler.reload(savedParser:java\.nio\.channels\.ReadableByteChannel)*]] where the argument is
```

Comment:
Honestly, I think these links that actually include the parameter types are pointless. What you want is a link to the right method, but there's no need for the method args to be displayed in the link text. Especially if the types are going to have their package names spelled out. It's just unreadable. It's much better to say "the other version of Compiler.reload" in prose than to have the method signature redundantly expressed here.

Reply:
Agreed. In order to have a link, scala requires parameters if there is ambiguity due to method overloading. However, one can provide an alternative text so that the parameters aren't shown. I'll go through all the scala links to make sure we're not showing excessive information. Javadoc seems to do a better job and making readable docs without much effort.